src/test/resources/pages/instructorFeedbackResultsSortRecipientGiverQuestion.html

```diff
@@ -3550,7 +3550,7 @@
           <tbody>
             <tr>
               <td>
-                <i>Instructors</i>
+                &lt;i&gt;Instructors&lt;/i&gt;
```

Comment:
Ah wait, @thyageshm somehow this line is processed as Text node. Is `<i>' a valid HTML tag?